tests/unittest/test_pubsub_crashpublish.py

```diff
@@ -90,7 +90,6 @@ def test_verify_topic_no_topic(self, client, pubsub):
         with pytest.raises(NotFound):
             client.rebuild_app(
                 {
-                    "LOCAL_DEV_ENV": "True",
```

Comment:
This is already specified in the `local_dev.env` file, so we don't need it here, too.